File: server/src/main/kotlin/models/pushable/PushableCompetitorInfo.kt
package models.pushable

data class PushableCompetitorInfo(
    val carIdx: Int,
    val userName: String,
    val userId: Int,
    val teamId: Int,
    val teamName: String,
    val carNumber: String,
    val carName: String,
    val carClass: Int,
    val carId: Int,
    val iRating: Int,
    val license: String,
    val licenseColor: Int,
    val carIsPaceCar: Int
)

File: logger/models/pushable/PushableCompetitorInfo.py
from dataclasses import dataclass


@dataclass
class PushableCompetitorInfo:
    carIdx: int
    userName: str
    userId: int
    teamId: int
    teamName: str
    carNumber: str
    carName: str
    carClass: int
    carId: int
    iRating: int
    license: str
    licenseColor: int
    carIsPaceCar: int
